test/end-to-end/pages/language/bilingualgrant.js

```diff
@@ -1,13 +1,14 @@
 'use strict';
 
-const commonContent = require('app/resources/en/translation/common');
+const contentEn = require('app/resources/en/translation/common');
+const contentCy = require('app/resources/cy/translation/common');
 
-module.exports = async function(answer) {
+module.exports = async function(language = 'en', answer) {
     const I = this;
+    const commonContent = language === 'en' ? contentEn : contentCy;
 
     await I.checkPageUrl('app/steps/ui/language');
     const locator = {css: `#bilingual${answer}`};
-    await I.waitForElement(locator);
```

Comment:
why would you want to remove this? this will help with stability, particularly in the overnight crossbrowser tests